### PR TITLE
feat: add time-master

### PR DESCRIPTION
**Pull request by MADAkwasi**

### CODE REVIEW

**Comment by fredrickyalley** [comment]
The alarm need to be placed beside the clock so when lots of alarm are added, you dont get a scrollable page.

**Comment by fredrickyalley** [comment]
If you miss an alarm, it doesnt ring again in let say 5mins after, if it hasn't been switched off

**Comment by fredrickyalley** [comment]
I think it should be possible to go either clockwise or anticlockwise when setting alarm time

File: week_2/time-master/function.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+export function renderAlarm(list, alarms) {
+	list.innerHTML = "";
+
+	const setAlarms = alarms
+		.map(
+			(alarm) =>
+				`
```
Comment (fredrickyalley) [inline week_2/time-master/function.js:9]: install a proper linter for code

File: week_2/time-master/style.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+}
+
+html {
+	font-size: 62.5%;
+}
+
+body {
+	background-color: #f1f0ff;
+	font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+	position: relative;
```
Comment (fredrickyalley) [inline week_2/time-master/style.css:16]: do you really need to set this?

**Comment by fredrickyalley** [comment]
Alarm keeps playing after you delete it

**Comment by MADAkwasi** [comment]
> Alarm keeps playing after you delete it

Done

File: week_2/time-master/style.css (hunk 2 above)
Comment (MADAkwasi) [reply week_2/time-master/style.css]: Done

File: week_2/time-master/function.js (hunk 1 above)
Comment (MADAkwasi) [reply week_2/time-master/function.js:9]: Edited but can you recommend one. I use Prettier

**Comment by MADAkwasi** [comment]
> If you miss an alarm, it doesnt ring again in let say 5mins after, if it hasn't been switched off

Done. It rings again after a minute

**Review by fredrickyalley — APPROVED** [review]

**Pull request merged**